### ISSUE TITLE
WipEout HD Fury [BCES00664]: Fury menu theme not displayed correctly

### ISSUE THREAD

**Issue by U99c** [RPCS3/rpcs3#8187]
Quick summary
The _Fury_ mode's main menu displays only slices of the ships instead of full representations.

Details
The expected behaviour of the Fury mode main menu theme is that it's supposed to play various sweeping angles of stylized 3D dot-point representations of the game's ships, along with swirly animations that clear the screen space and then set new impressions periodically.  

_Note: the Fury theme is the default as soon as the expansion pack is installed, but can be swapped via "options > game > menu style".  As the disc version comes with the expansion already, it's the default in this version._

Examples:
See how the menu is supposed to look like here: https://www.youtube.com/watch?v=I1s-eCBVedw

...and how the emulator shows it from 45 seconds here: https://www.youtube.com/watch?v=LbwgSwyJrbU 

Neither of these are my videos, but the respective behaviours are the same.  The RPCS3 version shows something that can't really be identified without know what it's supposed to be - and that's just the back half of a ship.  A slice of it, if you will.  

While on the other hand, as you can see, the official hardware version displays a full ship.  

Specs:
Windows 10 x64 up to date as per time of this post according to Win. Update, Ryzen 5 3600, 16GB 3200mhz RAM, GTX 1060 6GB driver ver. 445.75

Other notes:
1: This has been an issue for a long time, I've never seen it work correctly since I first tested the game over a year ago on an older i7 system.  

2: The issue is the same on both versions of the game I've tested, 2.00 & 2.30.  

[RPCS3.log](https://github.com/RPCS3/rpcs3/files/4608605/RPCS3.log)

**Comment by kd-11** [comment]
Please retest and update. From recent youtube videos the menu looks to have changed, not sure if it is correct or not.

**Comment by U99c** [comment]
Thanks for the response.  I just watched the menu for a few minutes, and noticed one ship in particular looked more complete than others, but it still did not render fully.  Most of the time, the ships looked the same as RPCS3 normally renders them.  

It looks like the ships are usually supposed to be built up section by section, from the rear moving forward - but only that first rear section is being displayed.

**Comment by AniLeo** [comment]
Needs updated information with screenshot comparisons on RPCS3 vs PS3

**Comment by U99c** [comment]
Pulled out my console to get some sustained example video, as there is still no change in how it's being rendered.  

Real PS3 footage of the screen (5 minutes): https://youtu.be/TbndQuU6kSk

RPCS3 footage (6 minutes-ish): https://youtu.be/H3VSFm00Hq8

Made a mistake while recording and didn't capture the settings, but I'm rather sure the only things that aren't default are the performance overlay and that the resolution is scaled to 4k.

As you can see, the menu running on the real PS3 generally comes off as being more dynamic and lively.   One notable example in the real footage, from about 1:34, you can see a ship be "built" from the dot matrix.  This never happens in the RPCS3 footage - the ships look like they're cut down.  

OpenGL shows no obvious difference to the Vulkan renderer.

**Comment by U99c** [comment]
Here's a log of the same version of RPCS3, showing the menu for a minute then loading a level: 

[RPCS3.log](https://github.com/RPCS3/rpcs3/files/5186954/RPCS3.log)

**Comment by Asinin3** [comment]
Please test graphical issues with 100% scaling as other values can cause problems. And since resolution scaling is a hack, we don't support tracking issues that are caused by it. I would be trying other settings too like write color buffers on, all the buffer settings in advanced tab on etc. Just experiment. And if it's still an issue upload a render doc of the issue. And it would be best to try and update the OP and keep this issue report concise/short.

* To create an RSX capture, use _Create_ _RSX_ _Capture_ under _Utilities_.
Captures will be stored in RPCS3 folder → captures.
	+ Compress your capture with 7z, Rar etc. 
	And drag and drop it into the issue.
	+ Or upload it to the cloud, such as Dropbox, Mega etc.
* To create a RenderDoc capture, please refer to [RenderDoc's documentation](https://renderdoc.org/docs/how/how_capture_frame.html).

**Comment by U99c** [comment]
[RPCS3.zip](https://github.com/RPCS3/rpcs3/files/5469432/RPCS3.zip)
[BCES00664_20201101033240_capture.zip](https://github.com/RPCS3/rpcs3/files/5469433/BCES00664_20201101033240_capture.zip)

My apologies, not sure how I managed to test it at 300% scaling, but in the case of this issue it doesn't seem to affect the results anyway.  

After producing the above log, I tested again with all your suggested buffers on with no other changes, as well as several combinations, ZCULL enabled/disabled, etc..  No meaningful difference.

**Comment by nitroos-ai** [comment]
I think this has been fixed here #11151.

**Comment by kd-11** [comment]
Closing as fixed

**Comment by kd-11** [comment]
Open a new issue. The cause is likely different.

**Comment by kd-11** [comment]
This ticket has been closed for years. Logs attached and captures are all useless now because the emulator has changed too much. Open a new issue with fresh logs and RSX capture so it can be investigated.

**Comment by U99c** [comment]
Hey! Sorry I didn't realise I could post in a closed topic otherwise I would have done this when I got notified it was 'fixed'.  Github noob!

This issue was never fixed.  

nitroos-ai said they thought it was, and the ticket was closed without verification the same day.  Also, he posted that less than 2 months ago, not years ago.  

I have periodically checked in to see how this game's performance has improved over time, and the menu background hasn't changed since I began.  The fix linked by nitroos-ai never did anything for Wipeout HD Fury.
